compiler/vm/core/src/shadowframes.c

```diff
@@ -1,6 +1,11 @@
 #include <robovm.h>
 
-void rvmPushShadowFrame(Env* env, ShadowFrame* frame) {
+#define LOG_TAG "shadowframes"
+
+void rvmPushShadowFrame(Env* env, void* functionAddress) {
+    ShadowFrame* frame = rvmAllocateMemory(env, sizeof(ShadowFrame));
```

Comment:
Yup, but all resolved.  Looks great, I suggest we merge this, can add llvm implementations of the methods at some point, but not required immediately. Id say strip out the existing llvm instruction injtection for the old stack shadow frame variable in the ShadowFramePlugin and we are good to go.